server/src/main/java/org/jboss/as/server/controller/resources/ServerRootResourceDefinition.java

```diff
@@ -444,6 +449,9 @@ public String getProductName() {
         // Paths
         resourceRegistration.registerSubModel(PathResourceDefinition.createSpecified(pathManager));
 
+        //capability registry
+        resourceRegistration.registerSubModel(new CapabilityRegistryResourceDefinition(capabilityRegistry));
+
```

Comment:
We need to work this out for the HC as well. It belongs under host=<localhostname>